content/20.data_objects.md

```diff
@@ -40,7 +40,7 @@ Additionally, by providing some routines as options, we can in some instances sp
 We demonstrate a handful of selection operations on a low-resolution dataset below.
 In Figure @fig:reg2 we illustrate the selection of a rectangular prism (i.e., a `region`, like in Section [sec:dobj-region].
 In Figure @fig:sp2, we illustrate the selection of a sphere (i.e., a `sphere`, like in Section [sec:dobj-sphere].
-And, to demonstrate the ability of yt to construct boolean selectors from these objects (i.e., Section [sec:dobj-bool] we show what the logical `NOT` of these two objects would produce in @fig:reg2_not_sp2.
+And, to demonstrate the yt's ability to construct boolean selectors from these objects (i.e., Section [sec:dobj-bool] we show what the logical `NOT` of these two objects would produce in @fig:reg2_not_sp2.
```

Comment:
You should remove "the" here:

"And, to demonstrate yt's ability to..."

Reply:
I thought I could apply the Batman rule here. Oh well, thanks :)